oracle/README.md

```diff
@@ -321,6 +337,79 @@ Verifies the database is available and accepting connections.
 
 ## Troubleshooting
 
+### Common Problems
+#### Oracle Instant Client
+- Verify that both the Oracle Instant Client and SDK files are located in the same directory.
+The structure of the directory should look similar:
+```text
+|____libociei.dylib
+|____libocci.dylib
+|____libocci.dylib.10.1
+|____adrci
+|____uidrvci
+|____libclntsh.dylib.19.1
+|____ojdbc8.jar
+|____BASIC_README
+|____liboramysql19.dylib
+|____libocijdbc19.dylib
+|____libocci.dylib.19.1
+|____libclntsh.dylib
+|____network
+|____xstreams.jar
+|____libclntsh.dylib.10.1
+|____libnnz19.dylib
+|____libclntshcore.dylib.19.1
+|____libocci.dylib.12.1
+|____libocci.dylib.18.1
+|____libclntsh.dylib.11.1
+|____sdk
+|____BASIC_LICENSE
+|____SDK_LICENSE
+|____libocci.dylib.11.1
+|____libclntsh.dylib.12.1
+|____libclntsh.dylib.18.1
+|____ucp.jar
+|____genezi
+|____SDK_README
+
+```
+##### Linux
+- See further installation documentation on [Oracle][15].
+
+##### Windows
+- Verify the Microsoft Visual Studio <YEAR> Redistributable requirement is met for your version, see [Windows downloads page][16] for more details.
+- Ensure the `PATH` environment variable contains the path of your directory (for example, `C:\oracle\`). 
+  Also verify that the environment variable `OCI_LIB64=C:\oracle` exists.
+
+
+#### JDBC Driver
+- If you encounter a `JVMNotFoundException`:
+
+    ```text
+    JVMNotFoundException("No JVM shared library file ({jpype._jvmfinder.JVMNotFoundException: No JVM shared library file (libjvm.so) found. Try setting up the JAVA_HOME environment variable properly.})"
+    ```
+
+    - Ensure that the `JAVA_HOME` environment variable is set and pointing to the correct directory.
```

Comment:
These instructions are for linux only but we seem to suggest ti applies to both  as we provide the command to test the result on windows as well. Can we split more clearly ? Maybe using tabs

Reply:
Seems like this comment still applies. `/etc/environment` is Linux specific

Reply:
We made a note that JDBC is Linux only in the configuration step

Reply:
There is a note just after for windows. There is no need to set the JAVA_HOME var on windows.

Reply:
I've removed the windows instructions since we don't recommend jdbc for windows now